Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Dockerfile for a standard freedom-for-node instance
+
+FROM selenium/node-node
+MAINTAINER Will Scott <willscott@gmail.com>
+
+USER root
+
+RUN apt-get update -qqy \
+  && apt-get -qqy install \
+    nodejs nodejs-legacy git npm 
```

Comment:
I'd expect a 'node-node' image to already have nodejs installed. why does it need to be re-installed?